clarity.d.ts

```diff
@@ -117,13 +117,15 @@ interface IShadowDomNode extends HTMLDivElement {
   ignore: boolean;  /* Flag to avoid sending data for that node */
 }
 
-interface ILayoutRectangle {
+interface IBoxModel {
   x: number; /* X coordinate of the element */
   y: number; /* Y coordinate of the element */
   width: number; /* Width of the element */
   height: number; /* Height of the element */
+  color: string; /* Color for visual aid */
   scrollX?: number; /* Scroll left of the element */
   scrollY?: number; /* Scroll top of the element */
+  overflow?: string; /* Overflow property for visualization aid */
```

Comment:
What does overflow mean though?

Reply:
It means that the parent element can hide any portions of the children that exceed its specified dimensions. This is a common technique used in CSS image spriting where an actual image contains a bunch of images and using CSS we control what's visible to the user. Without this box model will show up as incorrect.